src/System.Private.ServiceModel/tests/Common/Infrastructure/tests/TestPropertiesTest.cs

```diff
@@ -18,13 +18,7 @@ public static void TestProperties_Property_Names_Are_Initialized()
         // This test will fail to compile if these names are not generated.
         // It is not necessary to test all the known properties, just to
         // test that property name code generation occurred.
-        Assert.NotNull(TestProperties.BridgeResourceFolder_PropertyName);
-        Assert.NotNull(TestProperties.BridgeHost_PropertyName);
-        Assert.NotNull(TestProperties.BridgePort_PropertyName);
-        Assert.NotNull(TestProperties.UseFiddlerUrl_PropertyName);
-        Assert.NotNull(TestProperties.NegotiateTestDomain_PropertyName);
-        Assert.NotNull(TestProperties.NegotiateTestUserName_PropertyName);
-        Assert.NotNull(TestProperties.NegotiateTestPassword_PropertyName);
```

Comment:
Yes -- the NegotiateTestUserName and password no longer exist and needed to be removed, so I took the opportunity to prune out other obsolete entries.  This test was never intended to probe every TestProperty -- just probing one is enough to prove code-gen happened.